shop/src/main/java/ro/msg/learning/shop/model/Category.java

```diff
@@ -0,0 +1,29 @@
+package ro.msg.learning.shop.model;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
```

Comment:
Tie your imports, leave at most 1 empty line and no need for prefix or suffix empty lines within the functions/classes